osu.Game/Graphics/Containers/Markdown/OsuMarkdownFencedCodeBlock.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Syntax;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+
+namespace osu.Game.Graphics.Containers.Markdown
+{
+    public class OsuMarkdownFencedCodeBlock : MarkdownFencedCodeBlock
+    {
+        private Box background;
+        private MarkdownTextFlowContainer textFlow;
+
+        public OsuMarkdownFencedCodeBlock(FencedCodeBlock fencedCodeBlock)
+            : base(fencedCodeBlock)
+        {
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            // TODO : Change to monospace font to match with osu-web
+            background.Colour = colourProvider.Background6;
+            textFlow.Colour = colourProvider.Light1;
+        }
+
+        protected override Drawable CreateBackground()
+        {
+            return background = new Box
+            {
+                RelativeSizeAxes = Axes.Both,
+            };
+        }
+
+        public override MarkdownTextFlowContainer CreateTextFlow()
```

Comment:
Shouldn't this be using an `OsuMarkdownTextFlowContainer`?

Reply:
`base.CreateTextFlow()` is called `parentFlowComponent.CreateTextFlow()` to create the text flow.

`parentFlowComponent` itself is `OsuMarkdownContainer` and method `CreateTextFlow()` is already using `OsuMarkdownTextFlowContainer` in this PR.

https://github.com/ppy/osu/pull/12688/files/0d3ca8dde197428a2eef2ec2bc4d49d614ccea28#diff-757c2f47d90e82f18a6fa25089a827956e00ec82fdb35a05327d80902b378ce7R45